packages/waas/src/base.ts

```diff
@@ -34,7 +33,8 @@ import { SimpleNetwork, WithSimpleNetwork, toNetworkID } from './networks'
 type status = 'pending' | 'signed-in' | 'signed-out'
 
 const SEQUENCE_WAAS_WALLET_KEY = '@0xsequence.waas.wallet'
-const SEQUENCE_WAAS_SIGNER_KEY = '@0xsequence.waas.signer'
+const SEQUENCE_WAAS_SESSION_ID_KEY = '@0xsequence.waas.session_id'
+const SEQUENCE_WAAS_PAYLOAD_SIGNER_KEY = '@0xsequence.waas.payload_signer'
```

Comment:
lets rename to `SEQUENCE_WAAS_SESSION_SIGNER_KEY = '@0xsequence.waas.session_signer'` .. much more clear

Reply:
I think this will be removed when we use the ``IndexedDB``. I will apply if not.